include/rdma/fi_log.h

```diff
@@ -35,8 +35,6 @@
 #ifndef FI_LOG_H
 #define FI_LOG_H
 
-#include "config.h"
```

Comment:
why not? in case if user includes his config.h (or any other configuration header) prior to fi_log - then user will have standard way to enable debugging. alternative way is control ENABLE_DEBUG by command line (or other configuration options) to enable debuggins. in any case user is free to enable such option as he want